crypto/ocsp/ocsp_server.c

```diff
@@ -70,3 +70,149 @@ int OCSP_basic_add1_cert(OCSP_BASICRESP *resp, X509 *cert) {
   X509_up_ref(cert);
   return 1;
 }
+
+OCSP_SINGLERESP *OCSP_basic_add1_status(OCSP_BASICRESP *resp, OCSP_CERTID *cid,
+                                        int status, int revoked_reason,
+                                        ASN1_TIME *revoked_time,
+                                        ASN1_TIME *this_update,
+                                        ASN1_TIME *next_update) {
+  GUARD_PTR(resp);
```

Comment:
That this works is basically an implicit application of the C99 spec. Not very descriptive.

Reply:
I can change this back to a regular null check if we prefer that for this sort